andela/anbu-cfh
# Feedback reveiw 1

#### What does this PR do?
Make Header/Navigation bar consistent across all pages: Use icons for navigation menus. 
Make Avatars/Icons consistent across all pages
Make Avatars/icons be in sync with the images shown on the landing page.
Update App Favicon
Make Player cards in the gaming screen reflect the game theme better and follow a consistent color scheme.
Remove Numbered Icon and replaced with progress bar in game tour
Make buttons for the gaming tour consistent with the material design scheme for the app.
Use pastel colors  to allow it look seamless.
Make elements in the player avatar box make better use of the space available.
#### Description of Task to be completed?
Review feedback from demo day and apply changes.
#### How should this be manually tested?
Run the app using `npm start` or check the deployed version on Heroku and observe changes.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
